### DISCUSSION TITLE
windows shortcuts for powershell in linux (ubuntu)

### DISCUSSION THREAD

**Discussion by MohammadMD1383** [PowerShell/PowerShell#16539] [Q&A]
previously I had windows, right now I'm working with Ubuntu and have installed powershell from snap store.
I'm looking for windows behavior and shortcuts to use in linux powershell like:
ctrl + space -> menu completion (when I press it writes "@" why!?)
and many other shortcuts which were available in windows environment

so how can I have windows experience in linux powershell? thanks

**Comment by iSazonov** [comment]
I guess you could look PSReadline docs how configure your environment.

**Reply by MohammadMD1383** [reply]
Yes you're right.😅
but I have read them and tried to use `Set-PSReadLineKeyHandler` but that doesn't work.

**Reply by iSazonov** [reply]
The repo is more for pwsh developers. You could get more fast help in community resources like PSDiscord, forums and so on.
Or share more info about your trouble.
